Source/Csla.Analyzers/Csla.Analyzers/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "Csla.Analyzers": {
+      "commandName": "DebugRoslynComponent"
```

Comment:
that's allow debug of Analyzers, but not codeFix, you should add Analyzer project locally, but that at least half the way to do that nicely.